src/WebJobs.Script.WebHost/WebJobsScriptHostService.cs

```diff
@@ -200,9 +200,11 @@ private void CheckFileSystem()
                 // Check for marker file indicating a zip package failure, and if found stop the application.
                 // We never want to run with an incorrect file system.
                 string path = Path.Combine(_applicationHostOptions.CurrentValue.ScriptPath, ScriptConstants.RunFromPackageFailedFileName);
+
                 if (File.Exists(path))
                 {
-                    _logger.LogError($"Shutting down host due to presence of {path}");
+                    string errorMessage = File.ReadAllText(path);
```

Comment:
What happens if the `ReadAllText` throws any of the documented exception here? Do we have an outer catch which logs that exception?